aicssegmentation/structure_wrapper/WorkflowStep.py

```diff
@@ -8,7 +8,7 @@ class WorkflowStep:
     A class that defines a step in an AICS-Segmentation workflow.
     """
 
-    def __init__(self, step_config: Dict[str, str]):
+    def __init__(self, step_config: Dict[str, str], widget_info: Dict[str, Any]):
```

Comment:
Same comment as above - `widget_info` should probably be `function_info` or something along those lines